src/algorithms/reco/ScatteredElectronsEMinusPz.cc

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Daniel Brandenburg
+
+#include <Evaluator/DD4hepUnits.h>
+#include <Math/GenVector/LorentzVector.h>
+#include <Math/GenVector/PxPyPzM4D.h>
+#include <Math/Vector4Dfwd.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <edm4hep/Vector3f.h>
+#include <edm4hep/utils/vector_utils.h>
+#include <fmt/core.h>
+#include <math.h>
+#include <stdlib.h>
+#include <iterator>
+#include <map>
+#include <utility>
+
+#include "ScatteredElectronsEMinusPz.h"
```

Comment:
Please use full include paths relative to `src/` (future-proofing, uncheckable in CI).
```suggestion
#include "algorithms/reco/ScatteredElectronsEMinusPz.h"
```